deno/rest/v8/guild.ts

```diff
@@ -448,6 +448,8 @@ export interface RESTPatchAPICurrentGuildMemberNicknameJSONBody {
 
 /**
  * https://discord.com/developers/docs/resources/guild#modify-current-user-nick
+ *
+ * @deprecated Use [Modify Current Member](https://discord.com/developers/docs/resources/guild#modify-current-member) instead.
```

Comment:
Could you deprecate RESTPatchAPICurrentGuildMemberNicknameJSONBody too? and also add `RESTPatchAPICurrentGuildMemberJSONBody`?